lib/features/base/base_mailbox_controller.dart

```diff
@@ -553,4 +553,29 @@ abstract class BaseMailboxController extends BaseController {
     mailboxNode = personalMailboxTree.value.findNodeOnFirstLevel((node) => node.item.name?.name.toLowerCase() == name);
     return mailboxNode;
   }
+
+  void updateMailboxNameById(MailboxId mailboxId, MailboxName mailboxName) {
+    MailboxNode? selectedNode;
+
+    selectedNode = defaultMailboxTree.value.findNode((node) => node.item.id == mailboxId);
+    if (selectedNode != null) {
+      defaultMailboxTree.value.updateMailboxNameById(mailboxId, mailboxName);
+      defaultMailboxTree.refresh();
```

Comment:
Can you please indicate me what that `refresh` is doing?

Reply:
Refresh the UI. Nothing network related. [See here](https://pub.dev/documentation/get/latest/rx_types/RxObjectMixin/refresh.html)